tests/client/test_fileS.txt

```diff
@@ -0,0 +1,17 @@
+This is test file
```

Comment:
Use `dd` to create test_file while testing will be better.

test a 10MB file and a 100MB file will be enough.